src/fu-common.h

```diff
@@ -19,7 +19,9 @@ typedef enum {
 typedef enum {
 	FU_PATH_KIND_CACHEDIR_PKG,
 	FU_PATH_KIND_DATADIR_PKG,
+#ifdef EFI_APP_LOCATION
```

Comment:
If you do this then it changes the enum ordering depending on the compile settings, which would be uncool to debug. Just drop these and do the #else thing pls. Then squash, then push. Thanks.